### PR TITLE
Updated formatTimeStamp

### PR DESCRIPTION
**Pull request by divyjainjd**
-> Updated formatTimestamp function 
-> Added command to transpile ts files and minify js files in the version-bump workflow file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.0.9] - 2023-08-17
+
+### Changed
+
+- `formatTimestamp()` function will return time according to timezone of the user.
```
Comment (jainarpit014) [inline CHANGELOG.md:13]: Function will return formatted time as per Timezone passed in parameter. Previously IST was the default Timezone being used.

File: test/test.ts [hunk 2]
```diff
@@ -283,10 +283,10 @@ describe('getTextFromHtml', () => {
 });
 
 describe('formatTimestamp', () => {
-  it('should return the formatted date and time in IST', () => {
+  it('should return the formatted date and time according to region', () => {
```
Comment (jainarpit014) [inline test/test.ts:286]: according to timezone

File: .github/workflows/version_bump.yml [hunk 3]
```diff
@@ -27,6 +27,10 @@ jobs:
           with:
             node-version: 16
 
+        # Transpile the typescript files and minify javascript files
+        - name: Run minifyTs & minifyJs  
+          run: npm run production
```
Comment (jainarpit014) [inline .github/workflows/version_bump.yml:32]: These changes are no where mentioned in commit message of PR description. @divyjainjd

Comment (divyjainjd) [reply .github/workflows/version_bump.yml:32]: Added this changes in description @jainarpit014

File: CHANGELOG.md (hunk 1 above)
Comment (divyjainjd) [reply CHANGELOG.md:13]: @jainarpit014  we are not passing timezone in parameter. It is automatically taking it.

File: src/index.ts [hunk 4]
```diff
@@ -375,35 +375,33 @@ export const getTextFromHtml = (htmlString: string): string => {
 };
 
 /**
- * Formats a timestamp into a string representation of date and time in the format: DD-MM-YYYY HH:mm.
- * The timestamp is adjusted to the Indian Standard Time (IST) timezone.
+ * Formats a timestamp into a string representation of date and time according to region in the format: DD-MM-YYYY HH:mm.
+ * The timestamp is adjusted to the timezone of that region.
```
Comment (jainarpit014) [inline src/index.ts:379]: We should not mention words like region or timezone at all. It is only creating confusion. The function is simply convert 1 representation into another. @divyjainjd

Comment (divyjainjd) [reply src/index.ts:379]: Okay  @jainarpit014

Comment (divyjainjd) [reply src/index.ts]: Removed this @jainarpit014

File: test/test.ts [hunk 5]
```diff
@@ -283,10 +283,10 @@ describe('getTextFromHtml', () => {
 });
 
 describe('formatTimestamp', () => {
-  it('should return the formatted date and time in IST', () => {
+  it('should return the formatted date and time according to timezone', () => {
```
Comment (jainarpit014) [inline test/test.ts:286]: let's avoid using the word timezone or region anywhere for this function docs..this function is only converting unix timestamp to another format... @divyjainjd

Comment (divyjainjd) [reply test/test.ts]: @jainarpit014 updated the above for this functions docs

**Pull request merged**